drake/systems/plants/@Manipulator/parameterEstimation.m

```diff
@@ -25,20 +25,43 @@
 % @param data an instance of iddata (from the system identification 
 % toolbox; see 'help iddata' for more info) containing the data to 
 % be used for estimation.
-%
+% 
+% @option print_result determines if the function will print the results
+% 0 = no print, 1 = print estimated and original, 2 = only print estimated
+% 
+% @option method determines which method to use for estimation
+% 0 = no print, 1 = print estimated and original, 2 = only print estimated
+% 
+
+%% handle options
+if (nargin>2 && isstruct(varargin{1})) options=varargin{1};
+else options=struct(); end
+if (~isfield(options,'print_result')) 
+  options.print_result=0; 
+end
+if (~isfield(options,'method')) 
+  options.method=0; 
```

Comment:
you are still planning to add the method which does linear estimation of the identifiable parameters, then nonlinear least squares for the original params, right?